opsdroid/cli/start.py

```diff
@@ -9,20 +9,23 @@
 from opsdroid.core import OpsDroid
 from opsdroid.configuration import load_config_file
 from opsdroid.logging import configure_logging
-from opsdroid.const import DEFAULT_CONFIG_PATH
+from opsdroid.const import DEFAULT_CONFIG_LOCATIONS
 
 gettext.install("opsdroid")
 _LOGGER = logging.getLogger("opsdroid")
 
 
 @click.command()
-def start():
+@click.argument("path", required=False, type=click.Path(exists=True))
+def start(path):
     """Start the opsdroid bot."""
     check_dependencies()
 
-    config = load_config_file(
-        ["configuration.yaml", DEFAULT_CONFIG_PATH, "/etc/opsdroid/configuration.yaml"]
-    )
+    if path:
+        DEFAULT_CONFIG_LOCATIONS.insert(0, path)
+
+    config = load_config_file(DEFAULT_CONFIG_LOCATIONS)
+
```

Comment:
I would prefer that if a user sets the path flag we don't check the default locations at all. Because if they specify a bad path we will just move on to the default. This could be confusing because opsdroid would still start in this case.

Reply:
If the user gives a path that doesn't exist then click will throw an error and won't load. It will be something like:

```
Usage: __main__.py start [OPTIONS] [PATH]
Try "__main__.py start --help" for help.

Error: Invalid value for "[PATH]": Path "/Users/fabiorosado/Desktop/testt.yaml" does not exist.
```

With that in mind, should I just load the path that the user gives and ignore the rest of the default configs? 🤔

Reply:
Ah fair enough. Clever click!

Yeah I would just ignore the rest.